emissions.md

```diff
@@ -35,50 +47,65 @@ CO<sub>2</sub> emissions from gas flaring are not currently included in GCAM.
 
 ### <a name="co2-luc-emissions">CO<sub>2</sub> Emissions From Land-Use/Land-Use Change (LULUC)</a>
  
-Land-Use Change emissions are tracked separately. In general, emissions are estimated as:
+Land-Use Change emissions are tracked separately. See [Carbon Emissions](land.html#carbon-emissions).
 
-$$
-E_{r,t}=\Delta S_{r,t}=A_{r,t}*C_{r,t}-A_{r,t-1}*C_{r,t-1}
-$$
+### <a name="non-co2-overview">Non-CO<sub>2</sub> Emissions Overview</a>
 
-Where E indicates carbon emissions, S indicates carbon stocks, A indicates land area, and C indicates the average carbon density of the land area, across all land use types.
+We summarize here some general points common to non-CO<sub>2</sub> emissions in GCAM
 
-When land is converted to forests, the vegetation carbon content of that new forest land exponentially approaches an exogenously-specified, region-dependent value in order to represent the finite time required for forests to grow, as shown in the figure below.
 
-![Figure 1](gcam-figs/forest_carbon_evolution.png)<br/>
-Figure 1: Timescales for forest regrowth in GCAM.
-{: .fig}
+#### Initialization
 
-Changes in the carbon content of soils due to land-use change also exponentially approach an equilibrium value using a region-dependent timescale. This represents the timescales for carbon pool changes in soils.
+##### Data sources
 
-## <a name="non-co2-overview">Non-CO<sub>2</sub> Emissions Overview</a>
+* Non-CO<sub>2</sub> emissions, both GHGs & air pollutants in GCAM are initialized from the [CEDS inventory](https://github.com/JGCRI/CEDS-dev)([Hoesly et al 2018](#Hoesly2018)). Standard CEDS output datasets are used for input, although the emissions by GCAM regions are also generated as prebuilt data. Only the pre-built data  should be distributed publicly (to parallel with the IEA energy data). The data covers emissions for all GCAM regions from 1970 to 2019.  Only anthropogenic emissions (including open burning) are processed.
 
-We summarize here some general points common to non-CO<sub>2</sub> emissions in GCAM
+* The CEDS inventory do not contain emissions for grasslands, forest fires, deforestation and agricultural waste burning on fields. The data for these categories of emissions were added from the GFED LULC data set(as used in CMIP6). 
+
+* CEDS does not have a breakdown of emissions for road transport by mode. Therefore, GAINS emission factors by transport sectors (passenger, freight) and fuels to supplement the CEDS road emissions and derive emissions by different modes using emissions factors from the [GAINS data set](https://iiasa.ac.at/web/home/research/researchPrograms/air/ECLIPSEv5.html).
+
+* Additional information on fluorinated gases is from Guus Velders.
```

Comment:
Additional information on fluorinated gases is from 2019 EPA Global Non-CO2 Greenhouse Gas Emission Projection & Mitigation Potential Report [https://www.epa.gov/global-mitigation-non-co2-greenhouse-gases/global-non-co2-greenhouse-gas-emission-projections ](url)